modules/ob-disabling-shipwright-custom-resource.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
+[id="disabling-a-shipwright-build-resource_{context}"]
+= Disabling ShipwrightBuild resource
+
+You can disable the `ShipwrightBuilds` Custom Resource (CR) without disabling the shared-resources CSI driver component. Disabling `ShipwrightBuilds` CR removes the `ShipwrightBuilds` build controller and `ShipwrightBuilds` build webhook pod from the `openshift-builds` namespace.
```

Comment:
Would prefer something like "You can disable Shipwright Builds without disabling the Shared Resources CSI Driver component. Disabling Shipwright Builds...